CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -0,0 +1,31 @@
+# Changelog 0.1
+
+## 0.1.0 - 2022-06-16
+
+This is the first Theia release and it includes all Antrea flow visibility capabilities released as of 1.6.
+
+
+### Added
+
+- Helm support for Theia installation ([#15](https://github.com/antrea-io/theia/pull/15) [@yanjunz97]) ([#31](https://github.com/antrea-io/theia/pull/31) [@yanjunz97]) ([#34](https://github.com/antrea-io/theia/pull/34)  [@yanjunz97])
```

Comment:
Do we wanna mention the Helm support for Flow Aggregator installation in the Changelog?

Reply:
I deliberately did include it since it does not strictly pertain Theia